consumer-rest-core/src/main/scala/za/co/absa/spline/consumer/rest/controller/LineageDetailedController.scala

```diff
@@ -77,6 +78,20 @@ class LineageDetailedController @Autowired()(
     val dataSourceActionTypeOption = DataSourceActionType.findValueOf(access)
     repo.getDataSources(planId, dataSourceActionTypeOption)
   }
+
+  @GetMapping(value = Array("data-sources"))
```

Comment:
```suggestion
  @GetMapping(value = Array("execution-plans"))
```
We return execution plans details.